@AH-64D Apache Longbow/Addons/fza_ah64_systems/functions/interact/fn_interactBattSwitch.sqf

```diff
@@ -0,0 +1,40 @@
+/* ----------------------------------------------------------------------------
+Function: fza_systems_fnc_interactBattSwitch
+
+Description:
+	Sets button state for the APU sim.
+
+Parameters:
+	_heli   - The helicopter to get information from [Unit].
+
+Returns:
+	Whether to register a click (boolean).
+
+Examples:
+	...
+
+Author:
+	BradMick
+---------------------------------------------------------------------------- */
+params ["_heli"];
+
+private _battSwitchState = _heli getVariable "fza_systems_battSwitchState";
+
+switch (_battSwitchState
+) do {
+	case "OFF": {
+		_battSwitchState = "ON";
+		_heli setVariable ["fza_systems_battSwitchState", _battSwitchState, true];
+
+		false;
```

Comment:
Why should we not register a click in this scenario?

Reply:
Resolved